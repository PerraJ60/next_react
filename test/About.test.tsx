import TestRenderer from 'react-test-renderer';
import { MockedProvider } from '@apollo/client/testing';
//import About from 'pages/About';
import About, { ResumeQuery } from '../pages/About';

const mocks = [];

it('renders without error', () => {
  const component = TestRenderer.create(
    <MockedProvider mocks={mocks} addTypename={false}>
      <About />
    </MockedProvider>
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
