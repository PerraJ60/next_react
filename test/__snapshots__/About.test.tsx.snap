// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without error 1`] = `
<header
  className="header"
>
  <h1>
    Per Johansson
  </h1>
  <h2>
    loading...
  </h2>
</header>
`;
